examples/random_walk.py

```diff
@@ -0,0 +1,22 @@
+import time
+import random
+
+from ai2thor_wrapper.envs import ThorWrapperEnv
+
+if __name__ == '__main__':
+    # Random agent example with wrapper
+    max_episode_length = 10000
+    env = ThorWrapperEnv(max_episode_length=max_episode_length)
+    for episode in range(20):
+        start = time.time()
+        s = env.reset()
+        for t in range(max_episode_length):
```

Comment:
Fernando, the reason i have t+1 is because it looks weird to see 0-99

AND

if we did `range(1, max_episode_length)` that would be 1 less. It has to be:
`range(1, max_episode_length + 1)`
And now that's super confusing

Soooo. Stick with t+1?